e2e/groups/strengthened-approval-conditions.spec.ts

```diff
@@ -180,13 +183,37 @@ test(
       await groupSettingsPage.checkSuccessfulNotification();
     });
 
+    test.skip(true);
```

Comment:
I meant you replaced `test(...` by `test.skip(...`. But I assume that's because you do not want to skip all your super-long test. 
Yes, adding a comment would be nice with the reason + that it has to be removed + all the rest of test is skipped (I supposed that it is skipped from this line?).

Reply:
Yes, it will skip next lines. Fixed